packages/discovery-react-components/src/components/CIDocument/utils/__tests__/filterGroups.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { getBaseFilterGroups, FilterGroupWithFns } from '../filterGroups';
+
+describe('filterGroups', () => {
+  it('handles invalid ducment types correctly', () => {
```

Comment:
`ducment` -> `document`

Reply:
It's slang, Javier. Get with the times...

Reply:
I'm so old 😢

Reply:
lol--fixed!